translations/monero-core_pl.ts

```diff
@@ -1157,22 +1157,29 @@ The following information will be deleted
 
 The old wallet cache file will be renamed and can be restored later.
 </source>
-        <translation type="unfinished"></translation>
+        <translation>Czy jesteś pewien, że chcesz utworzyć od nowa pamięć podręczną portfela?
+Następujące informacje zostaną usunięte
+- Adresy odbiorców
+- Klucze transakcji
+- Opisy transakcji
+
+Poprzednia pamięć podręczna portfela zostanie zapisana pod inną nazwą i może być później przywrócona.
+</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="592"/>
         <source>Wallet log path: </source>
-        <translation type="unfinished"></translation>
+        <translation>Ścieżka loga portfela: </translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="596"/>
         <source>Wallet Name: </source>
-        <translation type="unfinished"></translation>
+        <translation>Nazwa portfela: </translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="600"/>
         <source>Daemon log path: </source>
-        <translation type="unfinished"></translation>
+        <translation>Ścieżka loga demona: </translation>
```

Comment:
In my opinion "Ścieżka logu demona" or even "Ścieżka dziennika demona" is more proper

Reply:
Yep. That's a good idea. I translated it too directly. I'll choose $2 option.